patches/ab_similar_recordings.py

```diff
@@ -0,0 +1,36 @@
+import troi.filters
+import troi.listenbrainz.recs
+import troi.musicbrainz.recording_lookup
+from troi.acousticbrainz.annoy import AnnoyLookupElement
+
+
+class ABSimilarRecordingsPatch(troi.patch.Patch):
+
+    @staticmethod
+    def inputs():
+        return [
+            {"type": str, "name": "recording_id", "desc": "MusicBrainz recording id", "optional": False},
+            {"type": str, "name": "similarity_type", "desc": "Type of acousticbrainz similarity", "optional": False}]
```

Comment:
It would be really good to list the possible similarities in the message or perhaps give a link to where they can be viewed.

Reply:
I was thinking about this. There's also a check here: https://github.com/metabrainz/troi-recommendation-playground/pull/11/files#diff-27abf483d83da8da4f40516af583a0e57706a1c0b345c06b22b54b2f5783073dR27, so it'd be great to be able to show it in the help without having to duplicate these constants. I'll see if argparse can help us here